plenoptic/simulate/canonical_computations/pooling_windows.py

```diff
@@ -0,0 +1,1051 @@
+"""contains the PoolingWindows class
```

Comment:
is all of this truly new?

Reply:
The `pooling_windows.py` file contains the `PoolingWindows` class -- that used to be in `pooling.py` but I split that file up because it was getting too big. So unfortunately it all got marked as "new".